src/mrnet/network/reaction_network.py

```diff
@@ -1594,6 +1632,162 @@ def mols_w_cuttoff(RN_pr_solved, cutoff=0, build_pruned_network=True):
+                                                                              1]) + "," + str(
+                    combined_products[0]) + "+" + str(
+                    combined_products[1])
+            elif len(combined_reactants) == 2 and len(combined_products) == 1:
+                node_str = str(combined_reactants[0]) + "+" + "PR_" + str(combined_reactants[
+                                                                        1]) + "," + str(
+                    combined_products[0])
+            elif len(combined_reactants) == 1 and len(combined_products) == 2:
+                node_str = str(combined_reactants[0]) + "," + str(
+                    combined_products[0]) + "+" + str(
+                    combined_products[1])
+            elif len(combined_reactants) == 1 and len(combined_products) == 1:
+                node_str = str(combined_reactants[0]) + "," + str(
+                    combined_products[0])
+            else:
+                node_str = None
+                print("Something is wrong", combined_reactants, combined_products)
+        return node_str
+
+
+    @staticmethod
+    def parse_reaction_node(node):
+        """
+        Method to parse out reactant,pr and prodcut nodes from a node string.
+        :param node: node string, ex "1+PR_2,3+4"
+        """
+        react_list = []
+        prod_list = []
+        if "PR" in node and "+" in node.split(",")[1]:
+            reactant = int(node.split("+PR_")[0])
+            pr = int(node.split("+PR_")[1].split(",")[0])
+            product1 = int(node.split("+PR_")[1].split(",")[1].split("+")[0])
+            product2 = int(node.split("+PR_")[1].split(",")[1].split("+")[1])
+            react_list = [reactant, pr]
+            prod_list = [product1, product2]
+            prod_list.sort()
+        elif "PR" in node and "+" not in node.split(",")[1]:
+            reactant = int(node.split("+PR_")[0])
+            pr = int(node.split("+PR_")[1].split(",")[0])
+            product = int(node.split("+PR_")[1].split(",")[1])
+            react_list = [reactant, pr]
+            prod_list = [product]
+        elif "PR" not in node and "+" in node:
+            reactant = int(node.split(",")[0])
+            product1 = int(node.split(",")[1].split("+")[0])
+            product2 = int(node.split(",")[1].split("+")[1])
+            prod_list = [product1, product2]
+            prod_list.sort()
+            react_list = [reactant]
+            prod_list = [product1, product2]
+            prod_list.sort()
+        elif "PR" not in node and "+" not in node:
+            reactant = int(node.split(",")[0])
+            product = int(node.split(",")[1])
+            react_list = [reactant]
+            prod_list = [product]
+        else:
+            print("Something wrong while parsing reaction node string", node)
+
+        return react_list, prod_list
+
+    @staticmethod
+    def identify_concerted_rxns_via_intermediates_NEW(RN, mols_to_keep=None,
```

Comment:
I'll ask around and see what other people's thoughts are; don't take this opinion as a final one.

That being said, my initial response is that naming a function NEW is a bad idea. This obviously has to have a different name, but if people start using this function, you don't want to make them switch to another one when you finally get rid of the old one. This one won't be NEW forever.

There are two options here. I think either are valid, though I have my preference:
1. Just get rid of the old one (preferred by me): we're still changing a LOT pretty constantly. Communicate - tell everyone what you're doing - but go ahead and break the old stuff. Carefully deprecating will be important once this code is beyond the alpha stage.
2. Add a deprecation warning (see what @mjwen did for the MolculeEntry methods).

Reply:
I agree that this should just replace the old method. My understanding is that the new approach is simply an iterative application of the old approach, so this feels like it can be backwards compatible, and just a strict improvement, though the user should have control over if they're doing one or two iterations of concerted reaction addition.